.github/workflows/ziggurat-release-event.yml

```diff
@@ -0,0 +1,35 @@
+name: ziggurat-release-event
+
+on:
+  repository_dispatch:
+    types: [ziggurat-release-event]
+
+jobs:
+  changelog-copy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Ziggurat repo
+        uses: actions/checkout@v2
+        with:
+          repository: shubhang93/ziggurat #TODO change it to gojek/ziggurat
+          token: ${{secrets.GH_TOKEN}}
+      - name: Copy changelog
+        run: |
+          cp ./CHANGELOG.md /tmp/CHANGELOG.md
+
+      - name: Checkout Ziggurat web repo
+        uses: actions/checkout@v2
+        with:
+          token: ${{secrets.GH_TOKEN}}
```

Comment:
shouldn't this be a `GITHUB_TOKEN` as explained here: https://docs.github.com/en/actions/reference/authentication-in-a-workflow

Same comment for #L15

And, probably we don't need github tokens to check out the code at lines 15 and 23. We might need it to make a commit at line 29. 

WDYT? Have we already verified this?

Reply:
So the GITHUB_TOKEN has limited permissions and to dispatch an event we require it on ziggurat's end, but might not be required on ziggurat web. Let me check it but we need a  token to commit and push